libs/cart/driver/magento/src/errors/transform.ts

```diff
@@ -1,7 +1,31 @@
+import { ApolloError } from 'apollo-client';
+
 import { daffTransformMagentoError } from '@daffodil/driver/magento';
+import { DaffCartDriverErrorMap } from '@daffodil/cart/driver';
+
+import { DaffCartMagentoErrorMap, DaffCartMagentoErrorMessageRegexMap } from './map';
+
+
+function transformMagentoCartGraphQlError(error: ApolloError): Error {
+	if (error.graphQLErrors.length) {
+    for (const code in DaffCartMagentoErrorMessageRegexMap) {
+      if (DaffCartMagentoErrorMessageRegexMap.hasOwnProperty(code)) {
```

Comment:
Won't the DaffCartMagentoErrorMessageRegexMap always has the code, since it's coming from the for-loop?

Reply:
Yeah this was required by the linter. https://stackoverflow.com/questions/44267094/tslint-complaining-statements-must-be-filtered-with-an-if-statement-when-using

Reply:
you can add `// tslint:disable-next-line` above line 11 instead

Reply:
done